packages/rainbowkit/src/wallets/walletConnectors/metaMaskWallet/metaMaskWallet.ts

```diff
@@ -33,6 +33,10 @@ function isMetaMask(ethereum: NonNullable<typeof window['ethereum']>) {
     return false;
   }
 
+  if (ethereum.isPortal) {
```

Comment:
Revert these changes, or provide additional documentation about the conflict. As long as the Ripio connector is available in the wallet list, this injected flag conflict should be automatically resolved.